doc/refman/RefMan-tac.tex

```diff
@@ -3507,6 +3515,12 @@ \subsection{\texorpdfstring{\texttt{cbn}}{cbn} and \texorpdfstring{\texttt{simpl
 tactic applications. It can result in expensive unifications, as
 refolding currently uses a potentially exponential heuristic.
 
+\begin{quote}
+  \optindex{Debug RAKAM}
+  {\tt Set Debug RAKAM}
+\end{quote}
+This option makes {\tt cbn} print various debugging information.
```

Comment:
Non-blocking: Might be nice to say what `RAKAM` stands for (r... a... k... abstract machine?)

Reply:
I guess @pirbo has the answer. Maybe [a tribute to Tintin](https://en.wikipedia.org/wiki/Red_Rackham%27s_Treasure)?

Reply:
My guess is `R` probably stands for refolding. The first `A` I have no idea, though. So @ppedrot's explanation is more plausible.

Reply:
`K` is for Krivine.

Reply:
Refolding Algebraic Krivine Abstract Machine

Reply:
@pirbo What does algebraic mean in this context?!

Reply:
It means that it is a machine that does `iota` in addition of `beta``delta`.
Said differently, it deals with algebraic datatypes (via case analyses/(co)fixpoint reduction) which the Krivine Abstract Machine has no idea about.

It comes from the (only on archive (and in my thesis)) description of the system:
- take a standard KAM
- add the machinery to deal with algebraic data structure (AKAM)
- add a second stack to deal with unfolded constant (RAKAM)

You obtain a imitation of `simpl` at 95% that have some simplicity/efficiency/customazability advantages over `simpl`  ( <-- argument contested), can be used to make unification refold while it reduced or be used in a strait forward way by `hnf` to make it refold